src/is/types.js

```diff
@@ -125,6 +125,22 @@ export function isWindowObject(value) {
     return value != null && typeof value === 'object' && 'setInterval' in value;
 }
 
+/**
```

Comment:
This should not be done here, but in `regexp.js` as mentioned in the issue. 
Checkout @boristane pull request #40